src/toggle/item/_toggle-item.scss

```diff
@@ -0,0 +1,38 @@
+.toggle-item {
+  --toggle-active-bg: white;
+  --toggle-active-color: darkgray;
+  --toggle-inactive-bg: black;
+  --toggle-inactive-color: white;
+  --toggle-disabled-bg: gray;
+  --toggle-disabled-color: white;
+  
+  list-style-type: none;
+
+  color: var(--toggle-inactive-color);
+  background-color: var(--toggle-inactive-bg);
+
+  &--is-selected {
+    color: var(--toggle-active-color);
+    background-color: var(--toggle-active-bg);
+  }
+
+  &--is-disabled {
+    color: var(--toggle-disabled-color);
+    background-color: var(--toggle-disabled-bg);
+
+    cursor: not-allowed;
+    
+    .list-item__click-wrapper {
+      pointer-events: none;
+    }
```

Comment:
With the latest changes, there wouldn't be this click wrapper div when isDisabled is true, would there ?

Reply:
Right, but we still need a child selector. Because the following hierarchy should be for display disabled cursor;
```.scss
.parent-selector {
  cursor: not-allowed;

    * {
      pointer-events: none;
    }
}
```
Can we use like that?